views/tx.tmpl

```diff
@@ -77,13 +79,9 @@
                     </tr>
                 {{end}}
                 {{if eq .Mature "True"}}
-                    <tr><td class="text-right pr-2 h1rem p03rem0">MATURE</td><td>{{.Mature}}</td></tr>
-                {{else}}
-                {{if and (eq .Confirmations 0) (or (eq .Type "Ticket") (eq .Type "Vote")) }}
-                    <tr><td class="text-right pr-2 h1rem p03rem0">MATURE</td><td>N/A</td></tr>
-                {{end}}
+                    <tr><td class="text-right pr-2 h1rem p03rem0">MATURITY</td><td>Fully Mature</td></tr>
```

Comment:
The phrase "Fully Mature" does not carry much meaning beyond just "mature" or "true", and it might be confusing (what is partially mature?).

Reply:
Yeah. I wasn't sure about that. But then again, there is a progress bar that can be partially full. Perhaps just a "Mature" row with no value that only shows when it is mature, i.e. drop the "True" from the current implementation.

Reply:
Perhaps, but it might look like there is a value missing.  You could post a screen shot and solicit opinions.  @xaur and @raedah might wanna comment.

Reply:
For labeling mature tickets/votes, I personally dislike "MATURITY Fully Mature" and just "MATURE ".  It looks a bit repetitive, but "MATURITY Mature" is clear and it is consistent with the "MATURITY {progress bar}" that precedes it, so that's my vote.

Reply:
Definitely not "Fully mature". Either "Maturity: \[progress bar\]", or "Mature: true|false" or "Maturity: Mature|Immature". Sorry I'm out of context, are we deciding progress bar vs text label?

Also this discussion does not belong to specific code review block, but rather the root PR or even the issue.

Reply:
Progress bar is now there for all immature tickets including unconfirmed mempool tickets. Just trying to work out what to do once it's mature. 

Sorry for the misplaced discussion. Should I re-comment in the related issue to elicit more feedback, or just go with "Maturity Mature"?